test/setup.js

```diff
@@ -1,41 +1,33 @@
-/*
- * Copyright © 2018 Lisk Foundation
- *
- * See the LICENSE file at the top-level directory of this distribution
- * for licensing information.
- *
- * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
- * no part of this software, including this file, may be copied, modified,
- * propagated, or distributed except according to the terms contained in the
- * LICENSE file.
- *
- * Removal or modification of this copyright notice is prohibited.
- */
 'use strict';
 
 require('colors');
+var chai = require('chai');
+var sinon = require('sinon');
+var sinonChai = require('sinon-chai');
 var supertest = require('supertest');
+process.env.NODE_ENV = 'test';
 
 // Root object
-var test = {};
+var testSuite = {};
```

Comment:
"Suite" to me sounds like a collection of tests. This is more like `testContext`/`testResources`/`testRun`/`testSetup` or something.